### PR TITLE
49 Code Coverage

### LINKED ISSUE #49: Code coverage report
We need coverage metrics in:
- [x] Travis
- [x] GitHub repo

### PR DESCRIPTION
**Pull request by mdingjan**
Will close #49

Reopening of #56 with a different base (dev instead of master).
Please have a look at the reference for other peer reviews.

### CODE REVIEW

File: tests/intern.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	// Unit test suite(s) to run in each browser
+	suites: [ /* 'myPackage/tests/foo', 'myPackage/tests/bar' */ ],
+
+	// Functional test suite(s) to execute against each browser once unit tests are completed
+	functionalSuites: [ /* 'myPackage/tests/functional' */ ],
+
+	// A regular expression matching URLs to files that should not be included in code coverage analysis. Set to `true`
+	// to completely disable code coverage.
+	excludeInstrumentation: /^(?:tests|node_modules)\//
+});
```
Comment (mpsijm) [inline tests/intern.js]: This file is not used, can you remove it?

Comment (mdingjan) [inline tests/intern.js]: I will, thanks for noticing! :)

**Comment by mpsijm** [comment]
Thanks for resolving my feedback, this is ready for merging :)

**Comment by thervh70** [comment]
I really like the new features. It will surely boost our code quality.

Will merge!

**Pull request merged**